ext/storage_sources/s3_store/s3_storage_source.cpp

```diff
@@ -370,157 +358,141 @@ S3Open(WT_FILE_SYSTEM *fileSystem, WT_SESSION *session, const char *name,
     return (0);
 }
 
-/*
- * S3Rename --
- *     POSIX rename, not supported for cloud objects.
- */
+// POSIX rename, not supported for cloud objects.
 static int
 S3Rename(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *from, const char *to,
   uint32_t flags)
 {
-    S3_STORAGE *s3 = FS2S3(file_system);
+    S3Storage *s3 = FS2S3(file_system);
 
-    (void)to;    /* unused */
-    (void)flags; /* unused */
+    UNUSED(to);
+    UNUSED(flags);
 
     s3->log->LogErrorMessage(std::string(from) + ": rename of file not supported");
     return (ENOTSUP);
 }
 
-/*
- * S3Remove --
- *     POSIX remove, not supported for cloud objects.
- */
+// POSIX remove, not supported for cloud objects.
 static int
 S3Remove(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name, uint32_t flags)
 {
-    S3_STORAGE *s3 = FS2S3(file_system);
+    S3Storage *s3 = FS2S3(file_system);
 
-    (void)flags; /* unused */
+    UNUSED(flags);
 
     s3->log->LogErrorMessage(std::string(name) + ": remove of file not supported");
     return (ENOTSUP);
 }
 
-/*
- * S3Size --
- *     Get the size of a file in bytes, by file name.
- */
+// S3ObjectSize --
```

Comment:
Let's be consistent in adding a function header. We have removed the header in other comments.